molecule/builder/Dockerfile

```diff
@@ -1,13 +1,33 @@
-ARG IMAGE
-FROM $IMAGE
+# April 28th, 2018
+FROM ubuntu@sha256:b8855dc848e2622653ab557d1ce2f4c34218a9380cceaa51ced85c5f3c8eb201
 
 # additional meta-data makes it easier to clean up, find
-ARG NAME
 LABEL org="Freedom of the Press"
-LABEL image_name="$NAME"
+LABEL image_name="trusty-sd-builder-app"
 
-# Build dependencies must not be added here, they are listed
-# in install_files/ansible-base/group_vars/all/securedrop
-RUN apt-get -y update && apt-get install -y python rsync sudo bash aptitude && \
-            aptitude -y dist-upgrade && \
-            apt-get clean
+RUN apt-get -y update && apt-get upgrade -y && apt-get install -y \
+        aptitude \
+        devscripts \
+        git \
+        gdb \
+        gnupg2 \
+        haveged \
+        libffi-dev \
+        libssl-dev \
+        make \
+        ntp \
+        inotify-tools \
+        paxctl \
+        python \
+        python-dev \
+        python-pip \
+        rsync \
+        ruby \
+        secure-delete \
+        sqlite \
+        unzip \
+        && apt-get clean \
+        && rm -rf /var/lib/apt/lists/*
+
+RUN paxctl -cm /usr/bin/python2.7
```

Comment:
Nit: You could make these all one `RUN` command to reduce the number of layers that are saved to the dev's disk.